pkg/transport_test.go

```diff
@@ -166,8 +221,28 @@ func TestConnectionClose(t *testing.T) {
 	resp, err := api.SetConfig(config)
 	assert.Nil(t, err)
 	assert.NotNil(t, resp)
-	err1 := api.Close()
+
+	httpApi := openapiart.NewApi()
+	httpApi.NewHttpTransport().SetLocation(httpServer.Location)
+	config1 := NewFullyPopulatedPrefixConfig(httpApi)
+	config1.SetResponse(openapiart.PrefixConfigResponse.STATUS_200)
+	resp1, err1 := httpApi.SetConfig(config1)
 	assert.Nil(t, err1)
+	assert.NotNil(t, resp1)
+
+	err2 := api.Close()
+	assert.Nil(t, err2)
+	data := NetStat("ESTABLISHED", t)
```

Comment:
can we avoid checking only for ESTABLISHED state ? a lingering tcp connection may be in non-established state too and still cause issues.

Reply:
updated